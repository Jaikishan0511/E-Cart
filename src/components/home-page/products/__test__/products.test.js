import React from "react";
import ProductContainer from "../index";
import ReactTestRenderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
const ProductContainerComp = (
    <Router>
      <ProductContainer />
    </Router>
  );
describe("Actions", () => {
    describe("Render", () => {
      it("should be a valid React Element", () => {
        expect(React.isValidElement(<ProductContainer />)).toBeTruthy();
      });
  
      it("should render without throwing an error", () => {
        expect(() =>
          ReactTestRenderer.create(ProductContainerComp )
        ).not.toThrow();
      });
    });
  
    describe("Snapshots", () => {
      it("matches the default snapshot", () => {
        const tree = ReactTestRenderer.create(ProductContainerComp ).toJSON();
        expect(tree).toMatchSnapshot();
      });
    });
  });