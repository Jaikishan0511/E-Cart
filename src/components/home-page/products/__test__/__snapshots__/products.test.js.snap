// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions Snapshots matches the default snapshot 1`] = `
<div
  className="Products-list"
/>
`;
